docs/source/docformat/index.rst

```diff
@@ -10,3 +10,11 @@ As an additional reference, small python packages demonstrates how docstrings ar
 
     epytext
     restructuredtext
+
+To override the default markup language for a module, define a module-level string variable 
+``__docformat__``, containing the name of the module's markup language.
+
+Language code is ignored and parser name is lowercased::
```

Comment:
I think the language code should also be case insensitive, yes.

Reply:
That sounds reasonable. So I think it would be good to already document that both the format and the language code are case insensitive, even though the language code isn't used for anything currently. That way, people who add `__docformat__` to their sources can already add both, if they have non-English docs.

We could actually use the language code already to produce a `lang=` attribute in HTML, which could help auto-translation of documents. But that would be better in a separate PR.